src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_SIZE = 10;
+    private Object[] keys;
+    private Object[] values;
```

Comment:
Use generic types here
```suggestion
    private K[] keys;
    private V[] values;
```